src/main/java/org/opensearch/flowframework/transport/GetWorkflowResponse.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.transport;
+
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.flowframework.model.WorkflowState;
+
+import java.io.IOException;
+
+/**
+ * Transport Response from getting a workflow status
+ */
+public class GetWorkflowResponse extends ActionResponse implements ToXContentObject {
```

Comment:
And lets change this class name to `GetWorkfowStateResponse`

Reply:
look at above [comments](https://github.com/opensearch-project/opensearch-ai-flow-framework/pull/148#discussion_r1391750164) regarding reasoning for not having GetWorkflowStateResponse as separate.